src/state/component-repository/components/actions.js

```diff
@@ -225,8 +225,15 @@ export const installECRComponent = (component, version) => dispatch => (
           dispatch(pollECRComponentInstallStatus(component.code))
             .then(res => resolve(res));
         } else {
-          dispatch(addErrors(data.errors.map(err => err.message)));
-          data.errors.forEach(err => dispatch(addToast(err.message, TOAST_ERROR)));
+          if (data && data.errors) {
```

Comment:
can you please remove that comment and make it more understandable? e.g. involving something like `const versionUnavailable = data && data.message` (if it's correct)